package.json

```diff
@@ -17,7 +17,6 @@
   "dependencies": {
     "@firebase/app": "^0.4.14",
     "@firebase/firestore": "^1.4.9",
-    "cheerio": "^0.22.0",
```

Comment:
Surprise dependency! 😂

Don't forget to add your name to the `contributors` section of this file!

Reply:
Contributed!